cmd/account/o-auth-enrollment/o-auth-enrollment.go

```diff
@@ -50,10 +49,8 @@ func newCreate() *cobra.Command {
 	cmd := &cobra.Command{}
 
 	var createReq oauth2.CreateOAuthEnrollment
-	var createJson flags.JsonFlag
 
 	// TODO: short flags
-	cmd.Flags().Var(&createJson, "json", `either inline JSON string or @path/to/file.json with request body`)
```

Comment:
This is a case of an API call that does take a JSON body where the JSON body disappears because it only contains optional and primitive fields. I suspect the suggestion in my comment at https://github.com/databricks/databricks-sdk-go/pull/666/files/939060619a6fd6ad4a7ab2e81fb103741571a1ea#diff-adb29e6ac8a45d7852be2e922930630dcb3ccd657e0ca0680a673b709e6017b2 fixes this.

Reply:
Address in Go SDK PR https://github.com/databricks/databricks-sdk-go/pull/666